api/v1beta1/glance_webhook.go

```diff
@@ -309,6 +320,11 @@ func (r *GlanceSpecCore) ValidateUpdate(old GlanceSpecCore, basePath *field.Path
 		// the same name. This represent a valid use case and we shouldn't prevent
 		// to grow the deployment
 		if _, found := old.GlanceAPIs[key]; !found {
+			// Fail if an invalid configuration/layout is detected for the current
```

Comment:
this part handles the situation where the CR exists and is update: if a new glanceAPI is added, we can't simply "continue" and allow any kind of configuration: we need to check if the backend is not valid (e.g. single + !file || split + file) and append an error to the APIs evaluation (that is a loop).